src/app/GetRecomendations/recomendationsListRenderer.js

```diff
@@ -0,0 +1,29 @@
+import { RecomendationListBuilder } from "./generateRecomendationsList";
+import { GenerateRecomendationsSourceList } from "../RecommendationsList/recomentadionsSource";
+import { PlaylistCreator } from "../Requests/PlaylistCreation/createPlaylistrunner";
+export class RecomendationsListRenderer {
+    static recomendationsDest = document.querySelector(".recomendations-list")
+    static recomendationsBg = document.querySelector(".recomendations-list__background")
+    static async renderRecomendationsList() {
+        let recommendationsObj = await RecomendationListBuilder.generateRecomendationsObj()
+        if (recommendationsObj == undefined) {
+            this.recomendationsDest.innerHTML = ''
+        }
+        else {
+            this.recomendationsBg.style.display = `flex`
+            let recomendationsHeader = `<h2>Recommendations list:</h2> <button class="recomendations-list__button" id="createPlaylist_button">Create playlist</button>`
+            let recomendationsListHTML = GenerateRecomendationsSourceList.generateListHTML(recommendationsObj, `p`)
+            this.recomendationsDest.innerHTML = recomendationsHeader + recomendationsListHTML
+            PlaylistCreator.createPlaylistBtnOn()
+        }
+    }
+
+    static lookforQuit() {
+        document.addEventListener("click", (evt) => {
+            let clidkedEl = evt.target;
+            if (clidkedEl == this.recomendationsBg) {
```

Comment:
If you use static class members, don't use this keyword, only use this keyword when using constructor

Reply:
this is basically event delegation by the way, you click host object and check for event target children instead of applying listeners to children